bool connect
---
int64 status
